apiserver/facades/client/client/backend.go

```diff
@@ -103,12 +103,13 @@ type Unit interface {
 // removed once all relevant methods are moved from state to model.
 type stateShim struct {
 	*state.State
-	model   *state.Model
-	session MongoSession
+	configSchemaSourceGetter config.ConfigSchemaSourceGetter
```

Comment:
You don't populate this. See `NewClient`

Reply:
Good spot!